### PR TITLE
fix: minor text tweakes

### PR DESCRIPTION
**Pull request by kilted-andres**
## fixes KILTProtocol/ticket#NoTicket

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ You have three different options:
 1. Using `yarn environment`:
 
    There is a script to facilitate the generation of the environment variables inside of this repository.
-   This script is called `./scripts/genesisEnvironmentVariables.ts`.
+   This script is called `./launchScripts/genesisEnvironmentVariables.ts`.
```
Comment (Dudleyneedham) [inline README.md:164]: Why is this changing to launch scripts?

Comment (Dudleyneedham) [reply README.md:164]: Why isn't it called scripts?

**Review by Dudleyneedham — COMMENTED** [review]
Why changing scripts? do we use scripts anywhere else in the project?

File: README.md (hunk 1 above)
Comment (kilted-andres) [reply README.md:164]: It is a remanence of last PR #82. I forgot to update the Reame there.

Comment (kilted-andres) [reply README.md:164]: it is just to make more clear that those scripts are just needed during the projects set-up.

**Review by aybarsayan — APPROVED** [review]
Looks good 😊

**Pull request merged**